semeio/jobs/scripts/gendata_rft.py

```diff
@@ -173,14 +173,18 @@ def main_entry_point():
     options = arg_parser.parse_args()
     logger.setLevel(options.log_level)
 
-    well_names = [w_info[0] for w_info in options.well_and_time_file]
-
-    trajectories = {
-        wname: Trajectory.load_from_file(
-            filepath=os.path.join(options.trajectory_path, wname + ".txt")
-        )
-        for wname in well_names
-    }
+    context_errors = []
+    trajectories = {}
+    for well_name, *_ in options.well_and_time_file:
+        try:
+            trajectories[well_name] = Trajectory.load_from_file(
+                filepath=os.path.join(options.trajectory_path, well_name + ".txt")
+            )
+        except (IOError, ValueError) as err:
+            context_errors.append(str(err))
+
+    if context_errors:
+        arg_parser.error("\n".join(context_errors))
```

Comment:
I am not convinced this is the right approach, but seems to be the one you prefer? I would rather just exit with the errors at this point, and not include the `argparse` `usage`, as that is just confusing in my mind, as this is a problem with the input, sure, but how does that `argparse` `usage` help?

Reply:
I suppose it was just to have errors on input be consistently handled through argparse (as in that will be what the user sees).

But I don't have that strong opinion really - the main point was more that one shouldn't throw the argparse error outside of the scope of argparse itself. As we are no longer in that scope I'm fine with sys.exit